contrib/forgejo/template.py

```diff
@@ -0,0 +1,54 @@
+pkgname = "forgejo"
+pkgver = "7.0.5"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_build_target = "all"
+make_check_target = "test-backend"
+make_use_env = True
+hostmakedepends = ["gmake", "go", "nodejs"]
+makedepends = ["linux-pam-devel"]
+depends = ["git", "git-lfs"]
+pkgdesc = "Git forge"
+maintainer = "triallax <triallax@tutanota.com>"
+license = "MIT"
+url = "https://forgejo.org"
+source = f"https://codeberg.org/forgejo/forgejo/archive/v{pkgver}.tar.gz"
+sha256 = "9b949f1f501911e278a709d78c885c411ad76ed1031888d106fae539086ce021"
+env = {
+    "EXTRA_GOFLAGS": "-trimpath -buildmode=pie",
```

Comment:
you probably want to append to this with self.get_goflags (and then you don't need the buildmode)

why trimpath isn't there and is instead in the util/ itself i have no idea..